src/commands/config/setClusterServer.ts

```diff
@@ -140,6 +153,9 @@ async function run(argv: Arguments) {
 
 export default createCommand({
   builder: yargs => {
+    const clusterOptions = yargs.argv.debug
```

Comment:
It would be better to `map()` from `clusterOptions` variable to obtain the ID list, instead of hard-coding the IDs in two places.